Dockerfile

```
@@ -1,163 +1,91 @@
-    librsvg2-dev \
-    fonts-liberation \
-    fonts-dejavu \
-    fontconfig \
-    && apt-get clean \
-    && rm -rf /var/lib/apt/lists/*
-
-# Copy package files first for effective caching
-COPY renderer/package*.json ./
-RUN npm ci
+# Install build tools
+RUN apk add --no-cache build-base git
```

Hadolint (2.12.0)

[warning] Pin versions in apk add. Instead of `apk add <package>` use `apk add <package>=<version>`

(DL3018)